WordPress/gutenberg
# Docs: List all minor versions on Versions in WordPress page

Alternative to #57910

## What?

This PR lists all WordPress minor versions on the [Versions in WordPress](https://developer.wordpress.org/block-editor/contributors/versions-in-wordpress/) page.

At the same time, for the first major version, I have removed the minor version number to match the notation in [the Release Archive](https://wordpress.org/download/releases/) page (Example: from `5.4.0` to `5.4`).

## Why?

In #57910, I suggested removing minor versions, but it seems to make more sense to list all minor versions.

## Testing Instructions

Compare [this PR document](https://github.com/WordPress/gutenberg/blob/docs/version-wordpress-minor/docs/contributors/versions-in-wordpress.md) and [the Release Archive page](https://wordpress.org/download/releases/) to make sure they are a complete match.
